pkg/apis/contrail/v1alpha1/tests/config_test.go

```diff
@@ -1783,3 +1792,17 @@ AUTHN_DOMAIN = Default
 cafile = /etc/ssl/certs/kubernetes/ca-bundle.crt
 ;AUTHN_TOKEN_URL = http://127.0.0.1:35357/v2.0/tokens
 `
+var configKeystoneAuthConf = `[KEYSTONE]
+#memcache_servers=localhost:11211
```

Comment:
Let's remove that commented code from template

Reply:
Done